main.go

```diff
@@ -124,14 +123,16 @@ func main() {
 
 	renewDeadline := time.Duration(leaderRenewSeconds) * time.Second
 	leaseDuration := time.Duration(int(1.2*float64(leaderRenewSeconds))) * time.Second
+	leaderRetryDuration := renewDeadline / 2
 
 	// Load controller manager configuration and create manager options from
 	// it.
 	serialExecutionStrategy := false
 	ctrlConfig := configstorageoscomv1.OperatorConfig{}
 	options := ctrl.Options{
 		Scheme:                     scheme,
-		LeaderElectionID:           "storageos-operator-leader",
+		LeaderElection:             true,
+		LeaderElectionID:           "storageos-operator-leader-imewimw",
```

Comment:
"imewimw" if we want to use a different name we could use something that made sense 😄